tools/pkglint/lib/rules.ts

```diff
@@ -131,7 +131,31 @@ export class NoticeFile extends ValidationRule {
   public readonly name = 'license/notice-file';
 
   public validate(pkg: PackageJson): void {
-    fileShouldBe(this.name, pkg, 'NOTICE', NOTICE);
+    fileShouldBeginWith(this.name, pkg, 'NOTICE', ...NOTICE.split('\n'));
+  }
+}
+
+/**
+ * NOTICE files must contain 3rd party attributions
+ */
+export class ThirdPartyAttributions extends ValidationRule {
```

Comment:
I think this should also check for:
- excessive attributes (e.g: we just removed/replaced a bundled dependency)
- we bundle transitive dependencies, too, so we need to attribute them, too...

And then I'm worried that this ensures that we have *some* attribution, but doesn't necessarily provide a mechanism to make sure we have *correct* attribution...